internal-include/darlingserver/process.hpp

```diff
@@ -170,6 +171,9 @@ namespace DarlingServer {
 		std::shared_ptr<Process> tracerProcess() const;
 		bool setTracerProcess(std::shared_ptr<Process> tracerProcess);
 
+		const std::string& executablePath() const;
+		void setExecutablePath(const char* path);
```

Comment:
It'd be more idiomatic for this to accept a `std::string`.
```suggestion
		void setExecutablePath(std::string path);
```

Reply:
Why `std::string` and not a `const std::string&`, we don't want to make that many copies.

Reply:
Ahh well using `std::string` with `std::move` results in the same number of copies.